### PR TITLE
Add taskschd

### PR DESCRIPTION
**Pull request by hcs64**

### CODE REVIEW

File: src/um/taskschd.rs [hunk 1]
```diff
@@ -0,0 +1,1158 @@
...
+        pPath: *mut BSTR,
+    ) -> HRESULT,
+    fn get_State(
+        pState: *mut TASK_STATE,
+    ) -> HRESULT,
+    fn get_CurrentAction(
+        pName: *mut BSTR,
+    ) -> HRESULT,
+    fn Stop(
+    ) -> HRESULT,
```
Comment (retep998) [inline src/um/taskschd.rs]: If the function has no parameters, it should be a single line.

**Review by retep998 — CHANGES_REQUESTED** [review]

File: src/um/taskschd.rs (hunk 1 above)
Comment (hcs64) [reply src/um/taskschd.rs]: done, thanks for the review!

**Review by retep998 — APPROVED** [review]

File: src/um/taskschd.rs [hunk 2]
```diff
@@ -0,0 +1,1237 @@
...
+    fn get_Item(
+        index: VARIANT,
+        ppRunningTask: *mut *mut IRunningTask,
+    ) -> HRESULT,
+    fn get__NewEnum(
+        ppEnum: *mut LPUNKNOWN,
+    ) -> HRESULT,
+}}
+
```
Comment (retep998) [inline src/um/taskschd.rs]: Evil blank line.

File: src/um/taskschd.rs [hunk 3]
```diff
@@ -0,0 +1,1237 @@
...
+        pDuration: BSTR,
+    ) -> HRESULT,
+    fn get_StopAtDurationEnd(
+        pStop: *mut VARIANT_BOOL,
+    ) -> HRESULT,
+    fn put_StopAtDurationEnd(
+        pStop: VARIANT_BOOL,
+    ) -> HRESULT,
+}}
+
```
Comment (retep998) [inline src/um/taskschd.rs]: Evil blank line.

**Review by retep998 — CHANGES_REQUESTED** [review]

**Review by retep998 — APPROVED** [review]

**Pull request merged**